time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -83,6 +83,7 @@ class TimeEntryCosmosDBModel(CosmosDBModel):
     technologies: List[str] = field(default_factory=list)
     end_date: str = field(default=None)
     deleted: str = field(default=None)
+    time_offset: str = field(default=None)
```

Comment:
Let's rename it to timezone_offset please

Reply:
Sure. I'll fix it.